scripts/dts/gen_defines.py

```diff
@@ -359,6 +362,29 @@ def write_compatibles(node):
             f"{node.z_path_id}_COMPAT_MATCHES_{str2ident(compat)}", 1)
 
 
+def write_child_identifiers(edt):
+    # Writes macro that lists identifiers of all children per node.
+    # Only enabled nodes are processed and identifiers for enabled child
+    # nodes are returned.
+    out_comment("Lists of children identifiers per node\n")
+    for node in sorted(edt.nodes, key=lambda node: node.dep_ordinal):
+
+        if node.enabled and node.children:
+            labels = ''
+            # For each subnode with non-empty list of labels add the
+            # first (non-alias) label to the string of ', ' separated
+            # list of labels.
+            for subnode in node.children.values():
+                if subnode.enabled and subnode.labels:
+                    labels = labels + subnode.labels[0]  + ', '
+
+            # Out list of labels only if non empty and remove trailing
+            # ', ' prior.
+            if labels:
+                out_dt_define(node.z_path_id + '_CHILD_NODELABELS',
```

Comment:
requires macros.bnf update

Reply:
Done

Reply:
Looks like the change from _CHILD_NODELABELS to _CHILDREN was only done for the API macro name -- could you do it for this too?

Reply:
ok